lib/retrieve-submission-data.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+'use strict'
+
+/* ::
+import type {
+  RetrievalData
+} from '../types.js'
+*/
+
+const axios = require('axios')
+const AWS = require('aws-sdk')
+
+function getCredentials (
+  formId /* : string */,
+  submissionId /* : string */,
+  jwt /* : string */
+) {
+  let api = axios.create({
+    baseURL: 'https://auth-api-test.blinkm.io',
```

Comment:
This should prb default to the production url and all users of the SDK (us) to pass a different variable. I would add this as another variable in the constructor of the Forms class

Reply:
I would suggest the parameter be called `oneBlinkAPIOrigin`